haystack/components/generators/hugging_face_local.py

```diff
@@ -51,6 +52,9 @@ def __init__(
         stop_words: Optional[List[str]] = None,
     ):
         """
+        Creates an instance of a HuggingFaceLocalGenerator. Unless specified otherwise in the `model`, the instance will
+        be initialized with the `google/flan-t5-base` model.
```

Comment:
```suggestion
        Creates an instance of a HuggingFaceLocalGenerator.
```
The default will be shown in the API docs.